src/components/patterns/itineraries/components/ItineraryStop.vue

```diff
@@ -158,7 +158,7 @@ export default {
 
     &__address {
         @include media-breakpoint-up(md) {
-            border-right: 1px solid $color-gray-tint-5;
+            // border-right: 1px solid $color-gray-tint-5;
```

Comment:
Can this be removed rather than commented out

Reply:
It needs to be put back in once the Opening Times put back in so wanted to keep it as a reminder to add it